examples/fc_demonstrate_artefact.py

```diff
@@ -0,0 +1,20 @@
+"""Demonstrate the artefact that can arise if
```

Comment:
I'm not sure it's really better, but maybe put the results in a table to get a nicer printout?
```python
"""
Demonstrate the artifact that can arise if
fc_fix_upper_and_lower_limit is not used.
"""
import numpy as np
from astropy.table import Table
from gammapy.stats import fc_find_acceptance_region_poisson

background = 3.5
cl = 0.90
x_bins = np.arange(0, 100)

table = Table()
table['mu'] = [0.745, 0.750, 0.755, 1.030, 1.035, 1.040, 1.045, 1.050, 1.055, 1.060, 1.065]
table['x_min'] = 0.0
table['x_max'] = 0.0

for row in table:
    x_min, x_max = fc_find_acceptance_region_poisson(row['mu'], background, x_bins, cl)
    row['x_min'] = x_min
    row['x_max'] = x_max

table.pprint()
```
gives the output:
```
  mu  x_min x_max
----- ----- -----
0.745   0.0   8.0
 0.75   1.0   8.0
0.755   1.0   8.0
 1.03   1.0   8.0
1.035   0.0   8.0
 1.04   0.0   8.0
1.045   0.0   8.0
 1.05   0.0   8.0
1.055   0.0   8.0
 1.06   1.0   9.0
1.065   1.0   9.0
```